bindings/java/src/org/sleuthkit/datamodel/ReadContentInputStream.java

```diff
@@ -39,6 +39,18 @@ public ReadContentInputStream(Content content) {
 		this.contentSize = content.getSize();
 	}
 
+	/**
+	 * Create a ReadContentInputStream that starts offset from the start of data
+	 *
+	 * @param content The Content object to read from
+	 * @param offset  The initial offset to start reading from
+	 */
+	public ReadContentInputStream(Content content, int offset) {
+		this.content = content;
+		this.currentOffset = offset;
+		this.contentSize = content.getSize();
+	}
+
```

Comment:
Could we satisfy the use case using the seek method instead of adding a new constructor?